compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/JvmAnnotationImplementationTransformer.kt

```diff
@@ -32,6 +34,20 @@ internal val annotationImplementationPhase = makeIrFilePhase<JvmBackendContext>(
 
 class JvmAnnotationImplementationTransformer(val jvmContext: JvmBackendContext, file: IrFile) :
     AnnotationImplementationTransformer(jvmContext, file) {
+    private val publicAnnotationImplementationClasses = mutableSetOf<IrClassSymbol>()
+
+    // FIXME: Copied from JvmSingleAbstractMethodLowering
```

Comment:
Before someone asks, the problem is that `allScopes` is protected in `IrElementTransformerVoidWithContext` which is in backend.common while `isInPublicInlineScope` is defined in backend.jvm, i.e., we can't define this utility as an extension function without refactoring somewhere. And I don't want to make such a refactoring part of this PR.